src/pybind/mgr/dashboard/tools.py

```diff
@@ -764,3 +766,31 @@ def str_to_bool(val):
     if isinstance(val, bool):
         return val
     return bool(strtobool(val))
+
+
+def restart_if_running():
+    """Restarts ceph-mgr iff CherryPy is serving."""
```

Comment:
s/iff/if/

Reply:
i think it's an abbreviation of "if and only if".